diff --git a/includes/plugins.inc b/includes/plugins.inc
index 0363fcb..24f7751 100644
--- a/includes/plugins.inc
+++ b/includes/plugins.inc
@@ -472,7 +472,7 @@ function ctools_plugin_load_includes($info, $filename = NULL) {
         }
         else {
 
-          require_once DRUPAL_ROOT . '/' . $file->uri;
+          include_once DRUPAL_ROOT . '/' . $file->uri;
           // .inc files have a special format for the hook identifier.
           // For example, 'foo.inc' in the module 'mogul' using the plugin
           // whose hook is named 'borg_type' should have a function named (deep breath)
